web/src/components/organisms/Project/settings/index.tsx

```diff
@@ -0,0 +1,188 @@
+import { ExclamationCircleOutlined } from "@ant-design/icons";
```

Comment:
Sorry I missed this. Please add this to the Icon atom and import from there!

Reply:
Are you sure we need to add all the icons to the atoms and re-export them? the number of used icons is very large, and they are directly related to antd.
I don't think we should do our own atom for it.